libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m

```diff
@@ -668,7 +670,7 @@ - (BOOL) webView:(UIWebView *) webView shouldStartLoadWithRequest:(NSURLRequest
          * If the request is attempting to refresh an invalid session, take over
          * the refresh process via the OAuth refresh flow in the container.
          */
-        NSString *refreshUrl = [self isLoginRedirectUrl:webView.request.URL];
+        NSString *refreshUrl = [self isLoginRedirectUrl:request.URL];
```

Comment:
In `UIWebView`, `request.URL` returns the correct value now, whereas `webView.request.URL` returns `nil`, since `webView.request` is `nil` (because it hasn't loaded the request yet).